challenges/objects-arrays.js

```diff
@@ -87,11 +91,19 @@ Name email@example.com
 
 Log the result of your new array. */
 const contactInfo = [];
+for(let i = 0; i < graduates.length; i++) {
+  contactInfo.push(`${graduates[i].first_name} ${graduates[i].email}`);
+}
 console.log(contactInfo);
 
 
 /* Request 3: Find out how many universities have the string "Uni" included in their name. Create a new array called uni that contains them all. Log the result. */
 const uni = [];
+for (let i = 0; i < graduates.length; i++) {
+  if (graduates[i].university.includes('Uni') === true) {
+    uni.push(graduates[i].university);
+  }
+}
 console.log(uni);
```

Comment:
There's an alternate solution to this problem by looping through the new universities array you made on line 80. See if you can play with that as well since it is fewer lines of code. Good work on finding the universities with 'uni'!